src/main/java/pl/vishop/vishopplugin/PendingTask.java

```diff
@@ -16,22 +16,22 @@
 
 package pl.vishop.vishopplugin;
 
+import java.util.List;
 import java.util.Set;
 import okhttp3.OkHttpClient;
 import org.bukkit.Bukkit;
-import org.bukkit.plugin.Plugin;
 import pl.vishop.vishopplugin.config.Config;
 import pl.vishop.vishopplugin.order.Order;
 import pl.vishop.vishopplugin.request.ConfirmRequester;
 import pl.vishop.vishopplugin.request.PendingRequester;
 
 public class PendingTask implements Runnable {
 
-    private final Plugin plugin;
+    private final ViShopPlugin plugin;
     private final OkHttpClient httpClient;
     private final Config config;
 
-    public PendingTask(final Plugin plugin, final OkHttpClient httpClient, final Config config) {
+    public PendingTask(final ViShopPlugin plugin, final OkHttpClient httpClient, final Config config) {
```

Comment:
Jeżeli potrzebujesz użyć "getBroadcastSender" to po wstrzyknij to konstruktorem tak jak reszta przekazanych tu rzeczy.